src/pages/public_registration/action_accordions/FindRegistration.tsx

```diff
@@ -0,0 +1,122 @@
+import { Registration } from '../../../types/registration.types';
+import { Autocomplete, Box, IconButton, TextField, Typography } from '@mui/material';
+import { dataTestId } from '../../../utils/dataTestIds';
+import SearchIcon from '@mui/icons-material/Search';
+import { useQuery } from '@tanstack/react-query';
+import { fetchResults } from '../../../api/searchApi';
+import { useState } from 'react';
+import { useDebounce } from '../../../utils/hooks/useDebounce';
+import { useTranslation } from 'react-i18next';
+import { ErrorBoundary } from '../../../components/ErrorBoundary';
+import { SearchListItem } from '../../../components/styled/Wrappers';
+import { RegistrationListItemContent } from '../../../components/RegistrationList';
+
+function isTitle(query: string) {
+  return !isDoi(query) && !isHandle(query);
+}
+
+function isDoi(query: string) {
+  return query.includes('https://doi.org/');
+}
+
+function isHandle(query: string) {
+  return query.includes('https://hdl.handle.net/');
+}
+
+interface FindRegistrationProps {
+  setSelectedRegistration: (registration: Registration | null) => void;
+  selectedRegistration: Registration | null;
+  filteredRegistrationIdentifier: string;
+}
+export const FindRegistration = ({
+  setSelectedRegistration,
+  selectedRegistration,
+  filteredRegistrationIdentifier,
+}: FindRegistrationProps) => {
+  const { t } = useTranslation();
+  const [searchBeforeDebounce, setSearchBeforeDebounce] = useState('');
+  const debouncedSearch = useDebounce(searchBeforeDebounce);
+
+  const fetchQuery = {
```

Comment:
fint å være eksplisitt på typen her, sånn at kopilator ser en feil om vi endrer search params en da, feks `title->name` e.l.
```suggestion
  const fetchQuery: FetchResultsParams = {
```